stats/stats/src/charts/mod.rs

```diff
@@ -128,23 +152,33 @@ pub trait ChartUpdater: Chart {
         &self,
         db: &DatabaseConnection,
         blockscout: &DatabaseConnection,
-        full: bool,
+        force_full: bool,
     ) -> Result<(), UpdateError> {
         let chart_id = crate::charts::find_chart(db, self.name())
             .await
             .map_err(UpdateError::StatsDB)?
             .ok_or_else(|| UpdateError::NotFound(self.name().into()))?;
-        let last_row = if full {
+        let min_blockscout_block = get_min_block_blockscout(blockscout)
```

Comment:
I would add log here, smth like tracing::info(min_blockscout_block = min_blockscout_block, "extracted min blockscout block for chart {}", self.name())